StortSpelProjekt/Engine/src/Renderer/ShadowInfo.cpp

```diff
@@ -103,11 +103,16 @@ void ShadowInfo::createResource(ID3D12Device5* device, unsigned int width, unsig
 
 void ShadowInfo::createDSV(ID3D12Device5* device, DescriptorHeap* dh_DSV)
 {
+	D3D12_DEPTH_STENCIL_VIEW_DESC dsvDesc = {};
+	dsvDesc.Format = DXGI_FORMAT_D24_UNORM_S8_UINT;
+	dsvDesc.ViewDimension = D3D12_DSV_DIMENSION_TEXTURE2D;
+	dsvDesc.Flags = D3D12_DSV_FLAG_NONE;
+
 	m_pDSV = new DepthStencilView(
 		device,
 		dh_DSV,
-		m_pResource,
-		DXGI_FORMAT_D24_UNORM_S8_UINT);
+		&dsvDesc,
+		m_pResource);
 }
```

Comment:
ShadowInfo should have a DepthStencil

Reply:
Again, its up to the developer. But the same here as in Bloom. The same resource is shared between several views, thats why I create 1 resource here, and 2 views separatly.